src/component/Note/Note.jsx

```diff
@@ -0,0 +1,108 @@
+import "./note.css";
+import { useState } from "react";
+import { useNote } from "../../context/NoteContext";
+
+const AddNote = () => {
+  const { addNotes } = useNote();
+  const [isNoteVisible, setIsNoteVisisble] = useState(false);
+  const initialState = {
+    title: "",
+    tags: "",
+    priority: "",
+    mainContent: "",
+  };
+  const [noteContent, setNoteContent] = useState(initialState);
```

Comment:
You can directly create object in this useState

Reply:
Actually i have passed the initial state like this bcz after clicking on button save i want my input form to go back to initial state.